### ISSUE TITLE
Launch Script Error

### ISSUE THREAD

**Issue by semecan** [crazy-max/nodejs-portable#28]
When I try to launch the "Shell" command I get the following error:
Error: open \\?\C:\Users\username\.nodejs-portable\launch.bat: The system cannot find the path specified.

I am guessing this is because I just downloaded the Windows binaries from https://nodejs.org/en/download/current/ and put them in the same folder as the nodejs-portable.exe.
So nodejs was not installed on my computer.
The batch file version of nodejs portable works perfectly fine.

**Comment by crazy-max** [comment]
Hi @semecan, please read the [support guidelines](https://github.com/crazy-max/nodejs-portable/blob/master/SUPPORT.md). Thanks

**Comment by xiazeyu** [comment]
I also met the problem.
It seems that you should create a folder at C:\Users\username\.nodejs-portable (replace username as your username).
You may deleted by accident.

**Comment by crazy-max** [comment]
@xiazeyu Can you post your logs plz?

**Comment by xiazeyu** [comment]
Here's when the error happen:

~~Seems nothing useful~~
```

2017-08-19 20:28:05 INFO - Starting Node.js Portable 2.1.0...
2017-08-19 20:28:05 INFO - Current path: J:\_apps\node
2017-08-19 20:28:05 INFO - >> SHELL
2017-08-19 20:28:05 INFO - Checking if Node.js installed... 
2017-08-19 20:28:05 INFO - Creating Node.js configuration... 
2017-08-19 20:28:05 INFO - Creating launch script... 
2017-08-19 20:28:05 ERRO - open \\?\C:\Users\xiaze\.nodejs-portable\launch.bat: The system cannot find the path specified.
2017-08-19 20:28:16 INFO - --------
2017-08-19 20:28:16 INFO - Starting Node.js Portable 2.1.0...
2017-08-19 20:28:16 INFO - Current path: J:\_apps\node
2017-08-19 20:28:16 INFO - >> SHELL
2017-08-19 20:28:16 INFO - Checking if Node.js installed... 
2017-08-19 20:28:16 INFO - Creating Node.js configuration... 
2017-08-19 20:28:16 INFO - Creating launch script... 
2017-08-19 20:28:16 ERRO - open \\?\C:\Users\xiaze\.nodejs-portable\launch.bat: The system cannot find the path specified.
2017-08-19 20:28:18 INFO - --------
2017-08-19 20:28:18 INFO - Starting Node.js Portable 2.1.0...
2017-08-19 20:28:18 INFO - Current path: J:\_apps\node
2017-08-19 20:28:18 INFO - >> SHELL
2017-08-19 20:28:18 INFO - Checking if Node.js installed... 
2017-08-19 20:28:18 INFO - Creating Node.js configuration... 
2017-08-19 20:28:18 INFO - Creating launch script... 
2017-08-19 20:28:18 ERRO - open \\?\C:\Users\xiaze\.nodejs-portable\launch.bat: The system cannot find the path specified.
2017-08-19 20:28:21 INFO - --------
2017-08-19 20:28:21 INFO - Starting Node.js Portable 2.1.0...
2017-08-19 20:28:21 INFO - Current path: J:\_apps\node
2017-08-19 20:28:21 INFO - >> SHELL
2017-08-19 20:28:21 INFO - Checking if Node.js installed... 
2017-08-19 20:28:21 INFO - Creating Node.js configuration... 
2017-08-19 20:28:21 INFO - Creating launch script... 
2017-08-19 20:28:21 ERRO - open \\?\C:\Users\xiaze\.nodejs-portable\launch.bat: The system cannot find the path specified.
2017-08-19 20:28:54 INFO - --------
2017-08-19 20:28:54 INFO - Starting Node.js Portable 2.1.0...
2017-08-19 20:28:54 INFO - Current path: J:\_apps\node
2017-08-19 20:28:54 INFO - >> SHELL
2017-08-19 20:28:54 INFO - Checking if Node.js installed... 
2017-08-19 20:28:54 INFO - Creating Node.js configuration... 
2017-08-19 20:28:54 INFO - Creating launch script... 
2017-08-19 20:28:54 ERRO - open \\?\C:\Users\xiaze\.nodejs-portable\launch.bat: The system cannot find the path specified.
2017-08-19 20:29:09 INFO - --------
2017-08-19 20:29:09 INFO - Starting Node.js Portable 2.1.0...
2017-08-19 20:29:09 INFO - Current path: J:\_apps\node
2017-08-19 20:29:09 INFO - >> SHELL
2017-08-19 20:29:09 INFO - Checking if Node.js installed... 
2017-08-19 20:29:09 INFO - Creating Node.js configuration... 
2017-08-19 20:29:09 INFO - Creating launch script... 
2017-08-19 20:29:09 ERRO - open \\?\C:\Users\xiaze\.nodejs-portable\launch.bat: The system cannot find the path specified.
```

There's when I executed `mkdir C:\Users\xiaze\.nodejs-portable`

```

2017-08-19 20:30:18 INFO - --------
2017-08-19 20:30:18 INFO - Starting Node.js Portable 2.1.0...
2017-08-19 20:30:18 INFO - Current path: J:\_apps\node
2017-08-19 20:30:18 INFO - >> SHELL
2017-08-19 20:30:18 INFO - Checking if Node.js installed... 
2017-08-19 20:30:18 INFO - Creating Node.js configuration... 
2017-08-19 20:30:18 INFO - Creating launch script... 
2017-08-19 20:30:18 INFO - Seeking Git path... 
2017-08-19 20:30:18 INFO - Git installation found: J:\_apps\PortableGit
2017-08-19 20:30:18 INFO - Adding node to path... 
2017-08-19 20:30:18 INFO - SETTING NODE_PATH... 
2017-08-19 20:30:18 INFO - Clearing screen...
2017-08-19 20:30:18 INFO - Starting up the shell... 
```

Then it can run properly.

**Comment by xiazeyu** [comment]
It happened at my system because I deleted C:\Users\xiaze\.nodejs-portable accidently.

**Comment by crazy-max** [comment]
Ok thanks for pointing this out!

**Comment by crazy-max** [comment]
Solved in 2.1.1
